lib/rdoc/generator/template/darkfish/css/rdoc.css

```diff
@@ -9,18 +9,26 @@
 /* vim: ft=css et sw=2 ts=2 sts=2 */
 /* Base Green is: #6C8C22 */
 
+/* 1. Variables and Root Styles */
 :root {
   --sidebar-width: 300px;
+  --primary-color: #2c8c5e;  /* A more muted green */
+  --secondary-color: #246b4b;  /* A darker, muted green */
+  --text-color: #333;
+  --background-color: #f8f9fa;
+  --link-color: #246b4b;  /* A muted teal-green color */
+  --link-hover-color: #25a28a;  /* A slightly brighter teal-green */
+  --border-color: #e0e0e0;
+  --sidebar-text-color: #2c3e50;  /* Dark blue-gray for contrast */
 }
 
-.hide { display: none !important; }
-
-* { padding: 0; margin: 0; }
-
+/* 2. Global Styles */
 body {
-  background: #fafafa;
-  font-family: Lato, sans-serif;
-  font-weight: 300;
+  background: var(--background-color);
+  font-family: 'Lato', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
```

Comment:
It might be better to pull this out to a variable, such as `--body-font-family`, so that it is easier to change font styles going forward, if desired.